README.md

```diff
@@ -19,19 +19,24 @@ module "admin_tier" {
   github_team                 = "${var.github_team}"
   instance_type               = "${var.instance_type}"
   vpc_id                      = "${var.vpc_id}"
-  security_groups             = ["${var.security_groups}"]
+  name                        = "admin"
+  namespace                   = "${var.namespace}"
+  stage                       = "${var.stage}"
   subnets                     = ["${var.subnets}"]
-  associate_public_ip_address = "${var.associate_public_ip_address}"
```

Comment:
@drama17 Yes, we need to fix `tf_instance`.
But in `README` it is good to see FULL example in order to do not confuse other people regarding module usage.
It is also necessary to update `Variables` section with `associate_public_ip_address` entry. And we should mention that `associate_public_ip_address` has `true` value by-default.